### PR TITLE
Add riscv architecture to api

### PR DESCRIPTION
**Pull request by adam-thorpe**
# Checklist

- [x] You added tests to cover the change
- [x] `mvn clean install` build and test completes
- [x] You changed or added to the documentation

Closes: https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/224

Signed-off-by: Adam Thorpe <adam.thorpe@ibm.com>

### CODE REVIEW

File: adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json [hunk 1]
```diff
@@ -250,6 +250,24 @@
...
+    },
+    {
+      "officialName": "Linux riscv64",
+      "searchableName": "RISCV64_LINUX",
+      "attributes": {
+        "heapSize": "normal",
+        "os": "linux",
+        "architecture": "riscv64"
+      },
+      "logo": "riscv.png",
```
Comment (adam-thorpe) [inline adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json:262]: Wasn't sure what this file does/should be

Comment (johnoliver) [reply adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json:262]: I am not sure this is even used anymore, probably fine

Comment (karianna) [reply adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json:262]: it is used in the images on the website when we display hero cards

Comment (gdams) [reply adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json:262]: yeah it's unused

**Pull request merged**